sorting.js

```diff
@@ -1,5 +1,12 @@
 function sortingCards(cardsDeck) {
-  return cardsDeck.sort();
-
-  // I couldn't think of a way to implement in Javascript and the one I made isn't working
+  for (var i = 0; i < cardsDeck.length; i++) {
+    for (var j = 0; j < cardsDeck.length - i - 1; j++) {
+      if (cardsDeck[j] > cardsDeck[j + 1]) {
+        var temp = cardsDeck[j];
+        cardsDeck[j] = cardsDeck[j + 1];
+        cardsDeck[j + 1] = temp;
+      }
+    }
+  }
+  console.log(cardsDeck);
```

Comment:
As mentioned before, no need to log the output, it should be returned so that it can be used in our program.